_posts/2016-05-20-tyxml4.md

```diff
@@ -5,7 +5,7 @@ author: The Ocsigen Team
 authorurl: https://ocsigen.org
 ---
 
-It is with great pleasure that we are announcing the release of [TyXML 4.0.0][changelog]. The new version is focused on improving the user experience for both old and new TyXML users.
+It is with great pleasure that we are announcing the release of [TyXML 4.0.0][changelog]. The major features of this new release are a new syntax extension that allows to use the standard HTML syntax and an improved user experience for both old and new TyXML users.
```

Comment:
That allows *using standard HTML syntax*. I would rephrase the part about the user experience, given that it is not a "concrete" feature. Something like:

> The major new features of this release is a new syntax extension that allows using standard HTML syntax. Various smaller improvements also contribute to an improved user experience, for old and new users alike.